yarn.lock

```diff
@@ -0,0 +1,7106 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
At least as far as the starter repo is concerned, I'd like to stick with npm as the default package manager rather than Yarn. The end user can obviously switch to yarn if they want, but that is a complication that the user needs to adopt for themselves. I don't want to risk any issues that might arise from having both yarn.lock and package-lock.json files in the template repository.

Reply:
That was an accident. I type "yarn" by reflex. I'm not committed to using yarn, and I kind of like nom better.

Reply:
Done